test/extension/image_processing/Jamfile

```diff
@@ -0,0 +1,19 @@
+# Boost.GIL (Generic Image Library) - tests
+#
+# Copyright 2020 Olzhas Zhumabek <anonymous.from.applecity@gmail.com>
+#
+# Use, modification and distribution are subject to the Boost Software License,
+# Version 1.0. (See accompanying file LICENSE_1_0.txt or copy at
+# http://www.boost.org/LICENSE_1_0.txt)
+#
+import testing ;
+
+compile anisotropic_diffusion.cpp ;
+compile hough_circle_transform.cpp ;
+compile hough_line_transform.cpp ;
+compile hough_parameter.cpp ;
+
+run anisotropic_diffusion.cpp ;
+run hough_circle_transform.cpp ;
+run hough_line_transform.cpp ;
+run hough_parameter.cpp ;
```

Comment:
Same here, repeating `compile` and `run` for the same tests is not necessary. 
- `compile` is for compile-only tests e.g. static asserts, nothing executed
- `run` is for tests you aim to build and execute